src/components/CourseForm/CourseForm.jsx

```diff
@@ -46,99 +56,37 @@ function CreateCourse(props) {
 	/**
 	 * Create a new author
 	 */
-	let createNewAuthor = () => {
+	const createNewAuthor = () => {
 		// Validation
 		let nameValidation = validation(
 			authorName.length >= 2,
 			'Author name length should be at least 2 characters'
 		);
 		if (!nameValidation) return;
 
-		// Copy the Main Authors list
-		let newAuthorsList = authorsList.slice();
-
 		// New author
 		let newAuthor = {
-			id: uuidv4(),
 			name: authorName,
 		};
 
-		// Add new author to the store
+		// Add new author to the store and server
 		props.createAuthor(newAuthor);
 
-		// Add this author to the Main list
-		newAuthorsList.push(newAuthor);
-
-		// Set new Main Authors list
-		setAuthorsList(newAuthorsList);
-
 		// Clear the input
-		document.getElementById('createAuthorName').value = '';
 		setAuthorName('');
 
 		return true;
 	};
 
-	/**
-	 * Add author to the course or remove
-	 *
-	 * @param {string} authorId
-	 * @param {boolean} moveToCourse
-	 */
-	let moveAuthor = (authorId, moveToCourse) => {
-		let aurhorIndex = 0;
-		let newAuthorsList = [];
-
-		if (moveToCourse) {
-			aurhorIndex = authorsList.findIndex((el) => el.id === authorId);
-
-			// Copy the Course Authors list
-			let newCourseAuthorsList = courseAuthorsList.slice();
-
-			// Add this author to the Course list
-			newCourseAuthorsList.push(authorsList[aurhorIndex]);
-
-			// Set new Course Authors list
-			setCourseAuthorsList(newCourseAuthorsList);
-
-			// Copy the Main Authors list
-			newAuthorsList = authorsList.slice();
-
-			// Remove this author from the Main list
-			newAuthorsList.splice(aurhorIndex, 1);
-		} else {
-			aurhorIndex = courseAuthorsList.findIndex((el) => el.id === authorId);
-
-			// Copy the Course Authors list
-			let newCourseAuthorsList = courseAuthorsList.slice();
-
-			// Remove this author from the Course list
-			newCourseAuthorsList.splice(aurhorIndex, 1);
-
-			// Set new Course Authors list
-			setCourseAuthorsList(newCourseAuthorsList);
-
-			// Copy the Main Authors list
-			newAuthorsList = authorsList.slice();
-
-			// Add this author to the Main list
-			newAuthorsList.push(courseAuthorsList[aurhorIndex]);
-		}
-
-		// Set new Main Authors list
-		setAuthorsList(newAuthorsList);
-	};
-
 	/**
 	 * Create a new course
 	 */
-	let createNewCourse = () => {
-		let id = uuidv4();
+	const createNewCourse = () => {
 		let date = new Date();
-		let creationDate = date.toLocaleDateString('en-US');
+		let creationDate = new Intl.DateTimeFormat('en-GB').format(date);
```

Comment:
The best way to send date to server is to use UTC date - optional

Reply:
The time is in the same format as in the Back-end.